components/settings/settings-panel.tsx

```
@@ -90,71 +90,72 @@ export function SettingsPanel() {
         <div className="space-y-2">
           {/* Saved Messages option */}
           <div
             className={`flex items-center justify-between p-3 rounded-xl border cursor-pointer transition-all ${
               storageChannelId === 'me'
-                ? 'border-[#2AABEE] bg-[rgba(42,171,238,0.1)]'
-                : 'border-[rgba(255,255,255,0.10)] hover:border-[rgba(255,255,255,0.22)] hover:bg-[#1C2733]'
+                ? 'border-[#DBDBDB]/60 bg-[#DBDBDB]/10'
+                : 'border-white/[0.15] bg-white/[0.05] hover:border-white/[0.30] hover:bg-white/[0.09]'
             }`}
             onClick={() => handleSelectChannel('me', 'Saved Messages')}
           >
```

Biome (2.4.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

(lint/a11y/useKeyWithClickEvents)